### PR TITLE
DOCS: Fix 2 failing URLs

### PR DESCRIPTION
**Pull request by VMWareJennyBeth**

### CODE REVIEW

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Upgrading PXF When You Upgrade Greenplum 6
 
 If you are using PXF in your current Greenplum Database 6.x installation, you must perform some PXF upgrade actions when you upgrade to a newer version of Greenplum Database 6.x. This procedure uses *PXF.from* to refer to your currently-installed PXF version.
 
-**Note**: if you are planning to upgrade a PXF `rpm` or `deb` installation, refer to the [PXF upgrade documentation](../../pxf/latest/upgrade_pxf_rpm.html).
+**Note**: if you are planning to upgrade a PXF `rpm` or `deb` installation, refer to the [PXF upgrade documentation](../../pxf/latest/upgrade_landing.html).
```
Comment (lisakowen) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:9]: this needs the /using/ as well.

**Review by lisakowen — APPROVED** [review]
thanks so much for fixing these!  just one minor comment.

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb [hunk 2]
```diff
@@ -58,11 +58,11 @@ After you upgrade to the new version of Greenplum Database, perform the followin
         gpadmin@gpmaster pxf cluster register
         ```
 
-    2. Start PXF on each segment host as described in [Starting PXF](../../pxf/latest/cfginitstart_pxf.html#start_pxf).
+    2. Start PXF on each segment host as described in [Starting PXF](../../pxf/latest/release/cfginitstart_pxf.html#start_pxf).
 
     3. Skip the following steps and exit this procedure.
 
-1. Starting in Greenplum Database version 6.19.0, PXF is removed from the Greenplum Server distribution. You must download and install the standalone PXF `rpm` or `deb` package as described in [Installing PXF](../../pxf/latest/installing_pxf.html). **Install the same PXF 5.x version as *PXF.from*.**
+1. Starting in Greenplum Database version 6.19.0, PXF is removed from the Greenplum Server distribution. You must download and install the standalone PXF `rpm` or `deb` package as described in [Installing PXF](../../pxf/release/installing_pxf.html). **Install the same PXF 5.x version as *PXF.from*.**
```
Comment (dyozie) [inline gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:65]: Shouldn't this one also be `pxf/latest/release` ?

**Review by dyozie — CHANGES_REQUESTED** [review]

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb (hunk 2 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:65]: Lisa said no.

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb (hunk 1 above)
Comment (VMWareJennyBeth) [reply gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb]: @lisakowen Does it?

Comment (lisakowen) [reply gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb]: yes, this needs /latest/using/.  (i could have sworn it was there ...)

File: gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb (hunk 2 above)
Comment (lisakowen) [reply gpdb-doc/markdown/pxf/upgrade_pxf_6x.html.md.erb:65]: yes, this needs /latest/release/.

**Pull request merged**